src/components/MapDisplay.js

```diff
@@ -55,15 +60,28 @@ const MapDisplay = withGoogleMap(({ searchResults, searchFromMap, mapPinFilter,
     return pin
   })
 
+  const clusterer = (
+    <MarkerClusterer
+      averageCenter
+      enableRetinaIcons
+      gridSize={25}
+      maxZoom={7}
+      >
+        {venuePins}
+    </MarkerClusterer>
+  );
```

Comment:
looks like you're using a ; on 72 but not on 61.  Or I haven't used them in so long I don't actually remember where they go.  Either way.